src/component/Links/Links.js

```diff
@@ -0,0 +1,113 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import Button from "@mui/material/Button";
+import MenuItem from "@mui/material/MenuItem";
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+
+const pages = [
+  "Home",
+  "About",
+  "Contact",
+  "Courses",
+  "Events",
+  <MoreHorizIcon/>
+ 
+];
+function Links() {
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+
+  const handleOpenNavMenu = (event) =>setAnchorElNav(event.currentTarget);
+  
+
+  const handleCloseNavMenu = () => setAnchorElNav(null);

```

Comment:
After formatting the file, please make sure to prevent the default click behavior on the buttons. Currently, the whole webpage is reloaded which goes against the single page application principles.

Reply:
whats wrong ??? don't workkk :(

Reply:
After we have discussed today, I expect you are capable of resolving the issues.

Reply:
You are not rendering the second box in case of desktop, and that's correct.
But you are still showing both Boxes in case of mobile:
![image](https://github.com/Leen-odeh12/Eduport_LMS/assets/81507185/d1d9dfa5-a5e1-4868-8734-249c8d994417)
This is because you have added: {isMobile && ...} but did not add the other condition to the second box, which is: {!isMobile && ...}
If you want, there is also a way of showing it using if(isMobile) Box1 else Box2